cmd/vcfFormat/vcfFormat.go

```diff
@@ -55,18 +72,20 @@ func usage() {
 	fmt.Print(
 		"vcfFormat - Options alter VCF formatting.\n" +
 			"Usage:\n" +
-			"vcfFormat input.vcf output.vcf\n" +
+			"vcfFormat input.vcf\n" +
 			"options:\n")
 	flag.PrintDefaults()
 }
 
 func main() {
-	var expectedNumArgs int = 2
+	var expectedNumArgs int = 1
+	var outfile *string = flag.String("o", "stdout", "Output file")
```

Comment:
I'm not sure I love the default stdout in most cases and would be curious to hear people's thoughts at code review. Certainly inconsistent with the style of much of the code base.